canonical-websites/www.ubuntu.com
# Add snap proxy to appstore

## Done

- Added Span Enterprise Proxy strip to /iot/appstore

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at the [appstore page](http://0.0.0.0:8001/internet-of-things/appstore):
- Compare to the [copy doc](https://docs.google.com/document/d/1CKwmPDDtXOIgw-jSZFwzP-qLMc_rS_dUACpHQXlg0fA/edit#)

## Issue / Card

Fixes #2846

## Screenshots

![image](https://user-images.githubusercontent.com/441217/37902568-b4a4d4c0-30ec-11e8-8871-be7c085c9d89.png)
